Scripts/Actuators/Thruster.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Laboratory for Underwater Systems and Technologies (LABUST)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Marus.Logger;
+using Marus.Utils;
+using UnityEngine;
+namespace Marus.Actuators
+{
+
+    public class Thruster : MonoBehaviour
+    {
+        public float lastForceRequest;
+        public float timeSinceForceRequest = 0.0f;
+        Rigidbody _vehicleBody;
+        Transform _vehicle;
+        GameObjectLogger<LogRecord> _logger;
+        public ThrusterAsset selectedThruster;
```

Comment:
Rename `selectedThruster` to `thrusterAsset`. Selected does not mean anything in this context